Maketargets.json

```diff
@@ -4,8 +4,8 @@
         "productnamelong": "syslog-ng Open Source Edition",
         "productnameshort": "syslog-ng OSE",
         "productnameinepic": "syslog-ng-open-source-edition",
-		"condition": "General.ose",
-        "conditions-to-exclude": "General.pe or General.pe6 or General.SPS or General.SRA",
+		"condition": "General.OSE",
```

Comment:
There are a lot of lowercase `MadCap:conditions="General.ose"` and ` MadCap:conditions="General.pe"` conditions in the docs. Should we change those as well?

Reply:
yes, I'll do this as well

Reply:
Oh cool, thank you. :)